### ISSUE TITLE
Empty page

### ISSUE THREAD

**Issue by ThisIsMyFavouriteJar** [u12206050/nova-treeview#1]
Hello, after following all installation steps I see an empty page.
So no categories are shown, and no errors are displayed.

Even after running npm install & npm run production this error occurs.

Any suggestions on what I am doing wrong? The package looks great in the screens,
but unfortunately I have not been able to use it yet.

**Comment by u12206050** [comment]
Did you register it properly in `/app/Providers/NovaServiceProvider.php`?

**Comment by ThisIsMyFavouriteJar** [comment]
Yes I did. I see the Categories link in the menu (in the "Trees" section), but when I click on it I get an empty page.

Here is my NovaServiceProvider:
![image](https://user-images.githubusercontent.com/25418904/90027794-38e7d780-dcb9-11ea-8978-f26e5835517d.png)

**Comment by u12206050** [comment]
What is the structure of your category tree?

**Comment by Chema-TM** [comment]
Same result, blank page.

I follow all steps to install, and not works

**Comment by u12206050** [comment]
Do you get any errors in the console or log file?

**Comment by Chema-TM** [comment]
No errors in console and no errors in log file.

Some information about the project:

```
- PHP 7.4.9
- Laravel 6.18.20
- Laravel Nova 2.12
```
I hope this information will be usefull for you.

thanks a lot for your help

**Comment by u12206050** [comment]
Hey again, thanks for the info, unfortunately I created this package using
Nova version 3, so not sure if that might have something to do with it.

Are you able to upgrade your Nova version?

**Comment by kvnblank** [comment]
Same issue with Laravel 7.25 and Nova 3.8.4.

**Comment by u12206050** [comment]
Hi, thanks to a pull request there seems to have been some issues with this package in production. Could you update to 0.5.1 and try again.

**Comment by ExsyDev** [comment]
@u12206050 is not working, the same result.

**Comment by u12206050** [comment]
@AbetaNL Have you solved this yet?
Unfortunately I a not actively involved in the project for which this library was created for, so I am working on it in my spare time.
PR are also welcome.
